suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -73,19 +73,22 @@ export const XpubScanScreen = ({
             networkType !== 'ethereum' &&
             isAddressValid(xpubAddress, networkSymbol)
         ) {
-            showAlert({
-                title: 'This is your receive address',
-                description: 'To check the balance of your coin, scan your public key (XPUB).',
-                icon: 'warningCircle',
-                pictogramVariant: 'red',
-                primaryButtonTitle: 'Got it',
-                onPressPrimaryButton: () => null,
-                secondaryButtonTitle: 'Where to find it?',
-                onPressSecondaryButton: () => {
-                    hideAlert();
-                    setIsHintSheetVisible(true);
-                },
-            });
+            // we need to set timeout to avoid showing alert during screen transition, otherwise it will freeze the app
+            setTimeout(() => {
```

Comment:
Works as temporary solution but I guess it's not really ideal to solve it with `setTimeout`

Reply:
Not sure what is better solution, probably not to use RN Modal component.